Sources/Kommunicate/Classes/URLBuilder.swift

```diff
@@ -14,7 +14,7 @@ final class URLBuilder {
     private var pathComponents = [String]()
 
     static var kommunicateBotsApi: URLBuilder {
-        return URLBuilder(host: "bots.kommunicate.io")
+        return URLBuilder(host: "api.kommunicate.io")
```

Comment:
Instead of renaming, we can remove `kommunicateBotsApi` and use `kommunicateApi` `var` for this new URL.